apps/game_backend/priv/repo/migrations/20240812135008_add_game_modes_table.exs

```diff
@@ -0,0 +1,13 @@
+defmodule GameBackend.Repo.Migrations.AddGameModesTable do
+  use Ecto.Migration
+
+  def change do
+    create table(:game_modes) do
```

Comment:
Why another table instead of an enum in `versions`?

Reply:
I see your point, but I think that marking a version as current in a game mode would be a bit more complex? What if we want to disabled a game mode? 🤔 . 
That's a good idea to be honest, but I would to see if this fits our necessities or it is too much 😆

Reply:
How do you envision to that now?